tomcat-gcp-lib/pom.xml

```diff
@@ -99,13 +99,21 @@
 
   <profiles>
     <profile>
-      <id>datastore-integration-test</id>
+      <id>gcloud-integration-test</id>
       <build>
         <plugins>
           <plugin>
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-failsafe-plugin</artifactId>
             <version>2.20</version>
+            <executions>
+              <execution>
+                <goals>
+                  <goal>integration-test</goal>
+                  <goal>verify</goal>
```

Comment:
The integration tests are run by default in the `verify` phase. I'm actually thinking we should get rid of the profile and instead document how to skip the integration tests with `mvn install -DskipITs`. WDYT?